ERROR_MESSAGES.md

```diff
@@ -12,16 +12,16 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 7 : CT - The measure reference results must have at least one measure
 * 8 : CT - A single aggregate count value is required
 * 9 : CT - Aggregate count value must be an integer
```

Comment:
Worth noting that this file is auto generated from ErrorCode.java. If you run `mvn clean package` that should regenerate this file with what ever updates you've made to ErrorCode.

Reply:
Yeah.. some of the validations were duplicates of other checks. When I removed the validations, the messages were no longer needed. I will run another `mvn clean install` to ensure my error messages are up to date.